docker/rockchip/Dockerfile

```diff
@@ -0,0 +1,97 @@
+# syntax=docker/dockerfile:1.6
+
+# https://askubuntu.com/questions/972516/debian-frontend-environment-variable
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG SLIM_BASE=debian:11-slim
+FROM ${SLIM_BASE} AS slim-base
+
+# Add wheels for RKNN-Toolkit-Lite2
+FROM wheels AS wheels-rk
+COPY docker/rockchip/requirements-wheels-rk.txt /requirements-wheels-rk.txt
+RUN pip3 wheel --wheel-dir=/wheels -r /requirements-wheels-rk.txt
+# ToDo: download RKNN-Toolkit-Lite2 wheel using requirements-wheels-rk.txt
+COPY docker/rockchip/rknn_toolkit_lite2-1.5.2-cp39-cp39-linux_aarch64.whl /wheels/
+
+# Collect deps in a single layer
+FROM deps-rootfs AS deps-rootfs-rk
+COPY docker/rockchip/yolov8n-320x320.rknn /models/
+# ToDo: Download librknnrt.so using wget; depends on RKNN-Toolkit-Lite2 wheel
+COPY docker/rockchip/librknnrt.so /usr/lib/
+
+# Frigate deps (ffmpeg, python, nginx, go2rtc, s6-overlay, etc)
+FROM slim-base AS deps
+ARG TARGETARCH
+
+ARG DEBIAN_FRONTEND
+# http://stackoverflow.com/questions/48162574/ddg#49462622
+ARG APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
+
+# https://github.com/NVIDIA/nvidia-docker/wiki/Installation-(Native-GPU-Support)
+ENV NVIDIA_VISIBLE_DEVICES=all
```

Comment:
can the rockcip support nvidia? seems unlikely and if so these should be removed

Reply:
No, I will remove it.